doc/swss-schema.md

```diff
@@ -147,6 +155,213 @@ and reflects the LAG ports into the redis under: `LAG_TABLE:<team0>:port`
     family        = "IPv4" / "IPv6"  ; address family
 
 
+---------
+##QOS related definitions
+
+###Definitions of common tokens
+	number					= 1*DIGIT
+	name					= number/1*ALPHA
+	ref_hash_key_reference	= "[" hash_key "]" ;Value the token is a refernce to the name of another valid DB key.
+	hash_key				= name ; a valid key name (i.e. exists in DB)
+
+###QUEUE_TABLE
+
+	; QUEUE table. Defines types of queues and their IDs to be referenced by the schedulers, ports.
+	; This is a 'template' for a queue[i], which then will be applied to a concrete port queue.
+	; For example can be applied to port.0.queue.3 port.1.queue.3, etc.
+	; SAI mapping - no direct mapping. As mentioned this is a 'template' for a queue.
+	key					= QUEUE_TABLE:name
+	;field
+	queue_index			= number
+	;field values
+	[0-7]
+	; Example
+	; key QUEUE_TABLE:BEST_EFFORT
+	;		field_name				value
+	; "queue_index"					  1
+	; key QUEUE_TABLE:RDMA
+	;		field_name				value
+	; "queue_index"					  4
+
+
+###TC\_TO\_QUEUE\_MAP\_TABLE
+	; TC to queue map
+	;SAI mapping - qos_map with SAI_QOS_MAP_ATTR_TYPE == SAI_QOS_MAP_TC_TO_QUEUE. See saiqosmaps.h
+	key					= "TC_TO_QUEUE_MAP_TABLE:"name
+	;field
+	tc_num				= number
+	;values
+	queue				= ref_hash_key_reference ;refernce to a queue hash key, see example below .
+	
+	; example
+	; key "TC_TO_QUEUE_MAP_TABLE:AZURE"
+	;	field_name					value
+	;		5					[QUEUE_TABLE:BEST_EFFORT]		; resolves to queue 1
+	;		6					[QUEUE_TABLE:BEST_EFFORT]		; resolves to queue 1
+
```

Comment:
I think that example should be a snapshot of a radis-cli output. It's more clear
(e.g. Why do you need "QUEUE_TABLE:BEST_EFFORT" and just use the queue index?